controller/state.go

```diff
@@ -453,6 +453,18 @@ func (m *appStateManager) CompareAppState(app *v1alpha1.Application, project *v1
 				LastTransitionTime: &now,
 			})
 		}
+
+		// If we reach this path, this means that a namespace has been both defined in Git, as well in the
+		// application's managedNamespaceMetadata. We want to emit a warning in such cases.
+		if isManagedNamespace(targetObj, app) {
+			conditions = append(conditions, v1alpha1.ApplicationCondition{
+				Type:               v1alpha1.ApplicationConditionRepeatedResourceWarning,
```

Comment:
@crenshaw-dev I reused `ApplicationConditionRepeatedResourceWarning`, let me know if I should create a new status or if this is good enough.